### DISCUSSION TITLE
Low Battery Icon

### DISCUSSION THREAD

**Discussion by Xaumir** [OnionUI/Onion#534] [Q&A] [answered]
Hi!

Is it possible for me to reenable the low battery icon ingame? Last night my console wasn't vibrating at all when the battery was getting low and it just kicked me out of the game I was playing back into the menu.
Thanks for the awesome work!

**Comment by Sichroteph** [comment, marked as answer]
Yes, it is on the to do list.

**Reply by Xaumir** [reply]
Nice, thank you!
